### PR TITLE
callData parse only on proxy sc

### PR DESCRIPTION
**Pull request by dragos-rebegea**

### CODE REVIEW

File: bridge-proxy/Cargo.toml [hunk 1]
```diff
@@ -23,18 +23,21 @@ path = "../common/tx-batch-module"
...
 [dependencies.multiversx-sc]
-version = "=0.51.1"
+version = "=0.52.2"
 
 [dependencies.multiversx-sc-modules]
-version = "0.51.1"
+version = "0.52.2"
 
-[dependencies.adder]
+[dependencies.crowdfunding-esdt]
```
Comment (dragos-rebegea) [inline bridge-proxy/Cargo.toml:35]: why do we have this dependency ?

Comment (CostinCarabas) [reply bridge-proxy/Cargo.toml:35]: For unit tests purposes (call a function from crowdfunding)

File: multisig/src/adder_proxy.rs [hunk 2]
```diff
@@ -0,0 +1,114 @@
+// Code generated by the multiversx-sc proxy generator. DO NOT EDIT.
+
+////////////////////////////////////////////////////
+////////////////// AUTO-GENERATED //////////////////
+////////////////////////////////////////////////////
+
+#![allow(dead_code)]
+#![allow(clippy::all)]
+
+use multiversx_sc::proxy_imports::*;
```
[the remark this answers is not in the document]
Comment (CostinCarabas) [reply multisig/src/adder_proxy.rs:10]: Replaced it with crowdfunding. Will remove adder.

File: bridge-proxy/tests/test-contract/crowdfunding-esdt.mxsc.json [hunk 3]
```diff
@@ -0,0 +1,196 @@
+{
+    "buildInfo": {
+        "rustc": {
+            "version": "1.80.0-nightly",
```
Comment (iulianpascalau) [inline bridge-proxy/tests/test-contract/crowdfunding-esdt.mxsc.json:4]: why nightly?

File: bridged-tokens-wrapper/src/esdt_safe_proxy.rs [hunk 4]
```diff
@@ -126,10 +126,10 @@ where
             .original_result()
     }
 
-    /// Converts failed Ethereum -> Elrond transactions to Elrond -> Ethereum transaction. 
+    /// Converts failed Ethereum -> MultiversX transactions to MultiversX -> Ethereum transaction. 
     /// This is done every now and then to refund the tokens. 
     ///  
-    /// As with normal Elrond -> Ethereum transactions, a part of the tokens will be 
+    /// As with normal MultiversX -> Ethereum transactions, a part of the tokens will be
```
Comment (iulianpascalau) [inline bridged-tokens-wrapper/src/esdt_safe_proxy.rs:132]: unfinished commment

File: multisig/tests/scenario_go_test.rs [hunk 5]
```diff
@@ -14,16 +14,19 @@ fn create_multiversx_to_ethereum_tx_batch_go() {
 }
 
 #[test]
+#[ignore] //Ignore for now
```
Comment (iulianpascalau) [inline multisig/tests/scenario_go_test.rs:17]: will these ignored tests be fixed?

File: multi-transfer-esdt/tests/scenario_go_test.rs [hunk 6]
```diff
@@ -5,26 +5,31 @@ fn world() -> ScenarioWorld {
 }
 
 #[test]
+#[ignore] //Ignore for now
```
Comment (iulianpascalau) [inline multi-transfer-esdt/tests/scenario_go_test.rs:8]: will these ignored tests be fixed?

File: bridge-proxy/tests/test-contract/crowdfunding-esdt.mxsc.json (hunk 3 above)
Comment (CostinCarabas) [reply bridge-proxy/tests/test-contract/crowdfunding-esdt.mxsc.json:4]: Fixed. Replaces with stable and latest framework (0.52.3).

File: multisig/tests/scenario_go_test.rs (hunk 5 above)
Comment (CostinCarabas) [reply multisig/tests/scenario_go_test.rs:17]: Yep, will fix these tests in a following PR.

**Review by iulianpascalau — APPROVED** [review]

**Review by CostinCarabas — APPROVED** [review]

**Pull request merged**